### PR TITLE
Adds tests for apps.get_model

### PR DESCRIPTION
**Pull request by gurkanindibay**
In issues #99 and #138, we had some problems when getting tenant column, when getting class with apps.get_model
Problem fixed wirh previous developments. I added a test to test the case

### CODE REVIEW

File: django_multitenant/tests/migrations/0024_data_load.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                ),
+                Account(
+                    name="pierredoe",
+                    domain="domain",
+                    subdomain="subdomain",
+                    country=Country.objects.get(name="France"),
+                ),
+            ]
+        )
+        country = Country.objects.get(name="USA")
```
Comment (agedemenli) [inline django_multitenant/tests/migrations/0024_data_load.py:65]: looks like an unused variable.
Maybe we can add an assertion for country count as well?

**Review by agedemenli — DISMISSED** [review]

**Comment by agedemenli** [comment]
there's a skipped test in the check details. As discussed offline, we'll create another issue for it. It's unrelated to this pr

File: django_multitenant/tests/migrations/0024_data_load.py (hunk 1 above)
Comment (gurkanindibay) [reply django_multitenant/tests/migrations/0024_data_load.py:65]: Added the assertion

**Review by agedemenli — DISMISSED** [review]

**Review by jonels-msft — COMMENTED** [review]
Two questions:

* The migration `reverse_func()` is uncovered by tests. While that seems potentially harmless, is it typical for the other test migrations? Or do we cover them by tests in some way?
* You're adding the test after the fix. Have you verified that the test fails prior to applying the fix? (Just want to be sure it's an effective test.)

**Comment by gurkanindibay** [comment]
> Two questions:
> 
> * The migration `reverse_func()` is uncovered by tests. While that seems potentially harmless, is it typical for the other test migrations? Or do we cover them by tests in some way?
> * You're adding the test after the fix. Have you verified that the test fails prior to applying the fix? (Just want to be sure it's an effective test.)

Removed reverse_func, 
Yes always verifying. When I'm working tests always fails, at the end I succeed to make them pass

**Pull request merged**